apps/web/components/dm/dm-channel-area.tsx

```
@@ -703,7 +702,15 @@ export function DMChannelArea({ channelId, currentUserId }: Props) {
     <div className="flex flex-col flex-1 overflow-hidden" style={{ background: "var(--app-bg-primary)" }}>
       {/* Header */}
       <div className="flex items-center gap-3 px-4 py-3 border-b flex-shrink-0" style={{ borderColor: "var(--theme-bg-tertiary)" }}>
-        <MobileMenuButton />
+        {/* Mobile: back arrow to DM list. Desktop: hidden (sidebar always visible). */}
+        <button
+          className="md:hidden w-8 h-8 flex items-center justify-center rounded transition-colors hover:bg-white/10"
+          style={{ color: "var(--theme-text-secondary)" }}
+          onClick={() => window.history.back()}
+          aria-label="Back to messages"
+        >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)